qt-models/diveplannermodel.cpp

```diff
@@ -894,8 +894,8 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 				add_dive_to_trip(copy, current_dive->divetrip);
 			record_dive(copy);
 			QString oldnotes(current_dive->notes);
-			if (oldnotes.indexOf(QString(disclaimer)) >= 0)
-				oldnotes.truncate(oldnotes.indexOf(QString(disclaimer)));
+			if (oldnotes.indexOf(QString(disclaimer).left(40)) >= 0)
```

Comment:
I do this. That's why in the actual truncation I compare to the actual new notes (displayed_dive.notes). This includes the markup.